client/app/scripts/superdesk-authoring/views/send-item.html

```diff
@@ -1,67 +1,73 @@
-                    <li ng-repeat="macro in macros">
-                        <button 
-                            ng-class="{active: macro.label === selectedMacro.label}" 
-                            ng-click="selectMacro(macro)">{{macro.label}}</button>
+            <div sd-toggle-box data-title="Embargo" data-open="true" ng-if="showEmbargo()">
+                <label class="publish-label" translate>Embargo</label>
+                <ul class="publish-list">
+                    <li>
+                        <div sd-datepicker ng-model="item.embargo_date" ng-disabled="!isEmbargoEditable()"></div>
+                    </li>
+                    <li>
+                        <div sd-timepicker ng-model="item.embargo_time" ng-disabled="!isEmbargoEditable()"></div>
                     </li>
                 </ul>
             </div>
-            <ul class="btn-list">
-                <li>
-                    <button class="cancel" ng-click="close()" translate>cancel</button>
+            <div sd-toggle-box data-title="Publishing" data-open="true" ng-if="showPublishSchedule()">
+                <label class="publish-label" translate>Publish Schedule</label>
+                <ul class="publish-list">
+                    <li>
+                        <div sd-datepicker ng-model="item.publish_schedule_date" ng-disabled="!_editable"></div>
+                    </li>
+                    <li>
+                        <div sd-timepicker ng-model="item.publish_schedule_time" ng-disabled="!_editable"></div>
+                    </li>
+                </ul>
+            </div>
+            <ul class="btn-list btn-bottom">
+                <li ng-if="mode === 'authoring' && canSendAndContinue()">
+                    <button class="send" ng-click="sendAndContinue()"
+                    ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
+                    translate>send and continue</button>
                 </li>
                 <li>
                     <button id="send-item-btn" class="send" ng-click="send()"
                     ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
                     translate>send</button>
                 </li>
-                <li ng-if="mode !== 'authoring'">
-                    <button class="send"
-                        ng-click="send(true)"
-                        ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage) || config"
-                        translate>send and open</button>
-                </li>
-                <li ng-if="mode === 'authoring' && canSendAndContinue()">
-                    <button class="send" ng-click="sendAndContinue()"
-                    ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
-                    translate>send and continue</button>
+                <li ng-if="mode === 'authoring'">
```

Comment:
My point is, shouldn't we change the label to correct if user is trying to correct an article or kill if user is trying to kill? Similar to what we are doing in `authoring-topbar.html`, at the moment. `publish()` function takes care of all the publishing actions mentioned earlier..